docs/UserGuide.md

```diff
@@ -461,34 +469,43 @@ Sets the weakness threshold for a weak marker.
 
 Format: `setweak g/GRADE`
 
-* By default, weakness threshold is set at Grade: C+
-* Updates weakness threshold value
-* Students with grade lower or equal to threshold grade appear with a marker in the UI
+* By default, weakness threshold is set at Grade: `C+`
+* The command updates the weakness threshold value.
+* Students with grade lower than or equal to the threshold grade appear with a marker in the UI. Look at the below image to see an example.
+  
+<img src="images/weak_label.png" alt="Weak Label" width="200">
+
 * `GRADE` should be one of the valid grades: **[A+, A, A-, B+, B, B-, C+, C, D+, D, F]**.
-* After setting the threshold, it will be maintained even after shutting down the application.
+* After setting the threshold with the command, the weak marker will be maintained even after shutting down the application.
+* Do note that the `setweak` command does not affect archived students. This means weak labels on archived students will not be changed.
```

Comment:
Since it is something to note, do you want to have it in a information box
```
<div markdown="block" class="alert alert-info">
**:information_source: Notes about setweak command:**<br>

* The `setweak` command does not affect archived students. This means weak labels on archived students will not be changed. <br>

</div>
```